peps/converters.py

```diff
@@ -199,6 +202,28 @@ def get_pep_page(artifact_path, pep_number, commit=True):
     return pep_page
 
 
+def get_commit_history_info(pep_filename):
+    commit_info = ''
+
+    try:
+        data = requests.get(
+            'https://api.github.com/repos/python/peps/commits?path={}'.format(pep_filename)
+        )
+    except Exception:
+        return commit_info
```

Comment:
```suggestion
    try:
        data = requests.get(
            'https://api.github.com/repos/python/peps/commits?path={}'.format(pep_filename)
        )
        data.raise_for_status()
    except Exception:
        return commit_info
```

It might be worthwhile to add `data.raise_for_status()`, to ensure that we always raise an exception here if we get an http error code like 404, 401, etc. Otherwise, it might raise a `JSONDecodeError` when we call `data.json()`. Realistically, I don't expect this will happen anytime soon to the github API under normal circumstances, but I think it would be a good general best practice to avoid making the PEP pages dependent on an external API returning a parse-able JSON response every time.

In general, I think it would also be better to catch `requests.exceptions.RequestException` instead of `Exception`, but I don't think it's an overly substantial concern in this case.